wagtail/models/__init__.py

```diff
@@ -1362,6 +1362,17 @@ def get_default_locale(self):
 
         return super().get_default_locale()
 
+    # set the default order for child pages to be sorted in wagtail
+    default_order = "-latest_revision_created_at"
```

Comment:
Just a small one - could we move this up alongside other class properties?

Maybe just after

```
max_count_per_parent = None
```